dvc/command/version.py

```diff
@@ -1,30 +1,97 @@
 from __future__ import unicode_literals
 
+import os
+import psutil
 import platform
 import argparse
 import logging
+import uuid
+from pathlib import Path
 
+from dvc.repo import Repo
 from dvc.command.base import CmdBaseNoRepo, append_doc_link
 from dvc.version import __version__
+from dvc.exceptions import DvcException, NotDvcRepoError
+from dvc.system import System
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdVersion(CmdBaseNoRepo):
     def run(self):
+        dvc_version = __version__
+        python_version = platform.python_version()
+        platform_type = platform.platform()
+
         info = (
             "DVC version: {dvc_version}\n"
             "Python version: {python_version}\n"
-            "Platform: {platform_type}"
+            "Platform: {platform_type}\n"
         ).format(
-            dvc_version=__version__,
-            python_version=platform.python_version(),
-            platform_type=platform.platform(),
+            dvc_version=dvc_version,
+            python_version=python_version,
+            platform_type=platform_type,
+        )
+
+        try:
+            root_directory = Repo.find_root()
+
+            info += ("Cache: {cache}\n").format(
+                cache=self.get_linktype_support_info()
+            )
+
```

Comment:
Btw, if we are inside of a dvc repo, let's also print fs type for the `repo.cache.local.cache_dir`. It is useful to see if workspace and cache are on the same fs/drive.